Fix `injectRoute` (#7943)
withastro/astro: .changeset/stupid-pants-press.md

---
'astro': patch
---

> Ensure that `injectRoute` is properly detected in monorepos and from node_modules

Reviewer: I guess this changeset would be more accurate now:

```suggestion
Ensure that injected routes from `node_modules` are properly detected
```
